packages/react-instantsearch/src/connectors/connectMenu.js

```diff
@@ -28,7 +28,7 @@ function getValue(name, props, searchState) {
   return name === currentRefinement ? '' : name;
 }
 
-const sortBy = ['count:desc', 'name:asc'];
+const sortBy = ['isRefined', 'count:desc', 'name:asc'];
```

Comment:
If it's a breaking change and we don't have a fallback, then I guess it's a no go for this change :) Maybe we could that change as a required change in #1734. What do you think @vvo ?